JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -239,7 +239,7 @@ - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
     self.toolbarHeightConstraint.constant = self.inputToolbar.preferredDefaultHeight;
-    [self.view layoutIfNeeded];
+    [self.view setNeedsLayout];
```

Comment:
```layoutIfNeeded``` was causing problems, I'm not sure why but sometimes (once in 3-4 screen openings) I was getting  self.topLayoutGuide.length == 0 (instead of 64) in the last ```-viewDidLayoutSubviews``` call and that caused wrong insets. If I call instead ```setNeedsLayout``` then the problem is gone. 

I tested a version without this line - looks fine so I'll remove it.